src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -199,4 +204,19 @@ private HearingWrapper createWrapper(HearingRequest hearingRequest) throws GetCa
                 .sscsServiceCode(sscsServiceCode)
                 .build();
     }
+
+    private void sendPartiesNotifiedUpdateRequest(HearingWrapper wrapper) {
+        hmcHearingPartiesNotifiedApi.updatePartiesNotifiedHearingRequest(
+                receiveIdamToken().getIdamOauth2Token(),
+                receiveIdamToken().getServiceAuthorization(),
+                getHearingId(wrapper),
+                getVersionNumber(wrapper),
+                buildUpdatePartiesNotifiedPayload(wrapper)
+
+        );
+    }
+
+    private IdamTokens receiveIdamToken() {
+        return idamService.getIdamTokens();
+    }
```

Comment:
This method is not needed, just use idamService.getIdamTokens(). ... instead

Reply:
Does it? I'm curious about this...I need to get better at LoD